src/main/java/org/csanchez/jenkins/plugins/kubernetes/ContainerReadinessProbe.java

```diff
@@ -0,0 +1,100 @@
+package org.csanchez.jenkins.plugins.kubernetes;
+
+import hudson.Extension;
+import hudson.model.AbstractDescribableImpl;
+import hudson.model.Descriptor;
+import org.jenkinsci.Symbol;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import java.io.Serializable;
+
+/**
+ * Created by geanpalacios on 15/02/23.
+ */
+public class ContainerReadinessProbe extends AbstractDescribableImpl<ContainerReadinessProbe> implements Serializable {
```

Comment:
Looks identical to `ContainerLivenessProbe`.

I would suggest to rename `ContainerLivenessProbe` to `ContainerProbe`, and add a compatibility alias as documented in http://www.jenkins.io/doc/developer/persistence/backward-compatibility/#rename-a-class